Applications/MIDIMonitor/SMMPreferencesWindowController.m

```diff
@@ -167,6 +163,12 @@ - (IBAction)changeExpertMode:(id)sender
     [self sendDisplayPreferenceChangedNotification];
 }
 
+- (IBAction)changeNewSourcesRadio:(id)sender
+{
+    [[NSUserDefaults standardUserDefaults] setInteger:[[sender selectedCell] tag] forKey:SMMAutoConnectNewSourcesPreferenceKey];
+    [self sendDisplayPreferenceChangedNotification];
```

Comment:
This isn't a "display preference" (it doesn't affect how messages are displayed) so you don't need to post this notification.